odoo/addons/base/models/assetsbundle.py

```diff
@@ -240,11 +252,11 @@ def clean_attachments(self, type):
             ('url', '=like', url),
             '!', ('url', '=like', self.get_asset_url(unique=self.version))
         ]
-
+        with without_unaccent(self.env.registry):
+            ira = ira.sudo().search(domain)
         # force bundle invalidation on other workers
         self.env['ir.qweb'].clear_caches()
-
-        return ira.sudo().search(domain).unlink()
+        return  ira.unlink()
```

Comment:
Minimal changes in order to reduce the posibility of conflicts

Reply:
This line of code is already as you suggested, what is the difference?

Reply:
Forget it

You are right!

Sorry